docs/content/Schema/Advanced/Code-Reusability-Export-and-Import.md

```diff
@@ -88,6 +88,31 @@ cube(`Events`, {
 });
 ```
 
+## Import from parent directories
+
+Cube.js flattens nested directories. Import from parent directories as they are located on the same level.
```

Comment:
```suggestion
Cube flattens nested directories. Import from parent directories as they are located on the same level.
```

Reply:
Also, are you trying to say you may need to import from parent directories as Cube flattens nested directories? 🙇

Reply:
@rpaik Yes, indeed, that might be a better wording for what I'm trying to say. Should I swap the whole paragraph?

Reply:
Yes, that works for me @dmitrika 😄

Reply:
Pushed updated paragraph version, please take another look.